fedgov-cv-ISDcreator-webapp/src/main/webapp/js/depositor.js

```diff
@@ -311,6 +313,15 @@ function createMessageJSON()
             "drivingLanes": drivingLaneArray
         };
 
+        if (approachArray[i].approachType === undefined) {
+            
+            for (let d = 0; d < drivingLaneArray.length; d++)
+                incompleteApproaches.push(drivingLaneArray[d].laneID);

```

Comment:
I guess using the first lane of the drivingLaneArray should catch the approach missing the approach type. No need to use for loop.